source/documentation/tools/tools-for-data-processing.md

```diff
@@ -0,0 +1,111 @@
+- **AP or Outside?** Does the code for the data processing need to be sent to another team or work area, and what tools do they have access to?
+
+## Processing data manually for exploration
+
+Below are the Analytical Platform tools typically used to manually undertake data processing during exploration. That is, you are trying stuff out and don’t need your code to be automated.
+
+### JupyterLab, RStudio or Visual Studio Code
+
+Typically, data users on the Analytical Platform will use [JupyterLab](https://user-guidance.analytical-platform.service.justice.gov.uk/tools/jupyterlab/index.html), [RStudio](https://user-guidance.analytical-platform.service.justice.gov.uk/tools/rstudio/index.html), or [Visual Studio Code](https://user-guidance.analytical-platform.service.justice.gov.uk/tools/visual-studio-code/index.html) to interact with and process data. There are several R or Python (or any other language using VSCode) packages you can use for this, and these tools integrate well with GitHub authentication for the Analytical Platform. So all you need to do is make sure you have access to the data itself (see [database access](https://user-guidance.analytical-platform.service.justice.gov.uk/tools/create-a-derived-table/database-access/) and [S3 warehouse data](https://user-guidance.analytical-platform.service.justice.gov.uk/data/amazon-s3/)).
+
+> ✅ Pros
+> *   [Version control through GitHub](https://user-guidance.analytical-platform.service.justice.gov.uk/github/) 
+> *   Data can be both structured (databases), and unstructured
+>*   Access to many packages, extensions and coding languages (especially if using Visual Studio Code)
+> *   Debugging through an Integrated Development Environment (IDE) such as Visual Studio Code
+> *   Easier to migrate code to [Airflow](https://user-guidance.analytical-platform.service.justice.gov.uk/tools/airflow/) (see below) or send to others to run if they have access to same tools
+
+> ❌ Cons
+> *   Initial setup time of projects is time-consuming
+> *   Code to run simple SQL is more complex than required on Athena workbench   
+
+### Amazon Athena Console
+
+Another option is to use the [Athena Console](https://user-guidance.analytical-platform.service.justice.gov.uk/data/curated-databases/amazon-athena/) directly on AWS through the Analytical Platform Control Panel. The Athena Console allows you to write and run SQL code directly. This is very handy when exploring data that is already in an Athena database and table.
+
+> ✅ Pros
+> * Quick to get started as you don’t need to install anything
+> * Creates modular SQL script thinking needed for [Create a Derived Table](https://user-guidance.analytical-platform.service.justice.gov.uk/tools/create-a-derived-table/) (see below)
+
+> ❌ Cons
+> * SQL only - limiting how complex your processing can be
+> * No proper version control, makes it hard for others to replicate
+> * Data has to be in an Athena database already 
+ 
```

Comment:
How about adding a section on create-a-derived-table? You don't have to use for orchestration, it also makes it easier to run one-off queries:

```suggestion
 ### Create-a-derived-table
Create-a-derived-table is an in-house developed service that brings  [dbt](https://docs.getdbt.com/docs/introduction), Git, and data access controls together to allow you to deploy derived and processed tables from data available on the Analytical Platform. This extends SQL with features commonly associated with programming languages, enabling more flexible, scalable, and maintainable code. dbt can also assure the quality of your transformations through [data tests](https://docs.getdbt.com/docs/build/data-tests), for example to check whether columns contain null values.
```